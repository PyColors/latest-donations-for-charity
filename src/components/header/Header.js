import React, { Component } from "react";
import Appbar from "muicss/lib/react/appbar";

class Header extends Component {
  render() {
    return (
      <Appbar className="app-bar">
        <header className="mui-appbar mui--z1">
          <div className="mui-container">
            <table>
             <tbody>
              <tr className="mui--appbar-height">
                <td className="mui--text-title">
                  <a href="https://www.macmillan.org.uk/" target="_blank">
                    <img
                      className="app-bar__brand-logo"
                      src="/src/images/macmillan-logo.jpg"
                      alt="macmillan"
                    />
                  </a>
                </td>
                <td className="mui--text-right">
                  <ul className="mui-list--inline mui--text-body2">
                    <li>
                      <a
                        className="app-bar__link"
                        href="https://github.com/PyColors/latest-donations-for-charity"
                        target="_blank"
                      >
                        Project on GitHub
                      </a>
                    </li>
                    <li>
                      <img
                        className="app-bar__img-build-travis"
                        src="https://travis-ci.org/PyColors/latest-donations-for-charity.svg?branch=master"
                        alt="Build Status"
                      />
                    </li>
                  </ul>
                </td>
              </tr>
               </tbody>
            </table>
          </div>
        </header>
      </Appbar>
    );
  }
}

export default Header;
