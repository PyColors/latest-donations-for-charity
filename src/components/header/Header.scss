.app-bar {
    &__brand-logo {
        width: 140px;
        position: relative;
        top: 4px;
    }

    &__link {
        color: white;
    }

    &__img-build-travis {
        position: relative;
        right: 3px;
        top: 4px;
    }
}
