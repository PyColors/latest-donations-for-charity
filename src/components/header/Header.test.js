import React from "react";
import Header from "./Header";

test("Render a div", () => {
  const wrapper = shallow(<Header>Hello Jest!</Header>);
  expect(wrapper).toMatchSnapshot();
});
