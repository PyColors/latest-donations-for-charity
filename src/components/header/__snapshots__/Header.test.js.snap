// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a div 1`] = `
<Appbar
  className="app-bar"
>
  <header
    className="mui-appbar mui--z1"
  >
    <div
      className="mui-container"
    >
      <table>
        <tbody>
          <tr
            className="mui--appbar-height"
          >
            <td
              className="mui--text-title"
            >
              <a
                href="https://www.macmillan.org.uk/"
                target="_blank"
              >
                <img
                  alt="macmillan"
                  className="app-bar__brand-logo"
                  src="images/macmillan-logo.jpg"
                />
              </a>
            </td>
            <td
              className="mui--text-right"
            >
              <ul
                className="mui-list--inline mui--text-body2"
              >
                <li>
                  <a
                    className="app-bar__link"
                    href="https://github.com/PyColors/latest-donations-for-charity"
                    target="_blank"
                  >
                    Project on GitHub
                  </a>
                </li>
                <li>
                  <img
                    alt="Build Status"
                    className="app-bar__img-build-travis"
                    src="https://travis-ci.org/PyColors/latest-donations-for-charity.svg?branch=master"
                  />
                </li>
              </ul>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </header>
</Appbar>
`;
